<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButtonQ">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonW">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonE">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonR">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonT">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonY">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonU">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonI">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonO">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonP">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonA">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonS">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonF">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonG">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonH">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonJ">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonK">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonL">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonZ">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonX">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonC">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonV">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonN">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonM">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>350</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Dica:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resultado:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vidas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Palavra Secreta</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
